.github/workflows/release.yml

```diff
@@ -84,6 +84,10 @@ jobs:
         run: protocol-models/bin/generate-typescript-classes-docker.sh
 
       # github runners already come with NPM and node
-      - run: cd protocol-models/typescript && npm publish
+      - name: Generate Typescript Protocol Classes
```

Comment:
every step needs to have this if block added to it 
```
if: ${{ steps.release.outputs.release_created }}
```
for example: https://github.com/airbytehq/airbyte-protocol/blob/main/.github/workflows/release.yml#L55

I will refactor this workflow to encapsulate all the release steps to be in a separate job with only one if statement. I'm not sure what possessed me to do it this way in the first place. 

This conditional is needed because a release is only made when the release PR is merged and should not be done for ever merge to main.

Reply:
👍